src/Landing/Hero/HeroLayout.tsx

```diff
@@ -22,26 +21,32 @@ const StyledHeroLayout = styled('section')((props) => {
   };
 });
 
-const StyledBackground = styled(Box)({
-  position: 'absolute',
-  left: 0,
-  right: 0,
-  top: 0,
-  bottom: 0,
-  backgroundSize: 'cover',
-  backgroundRepeat: 'no-repeat',
-  zIndex: -2,
-});
-
-type HeroLayoutProps = {
-  sxBackground: SxProps<Theme>;
+type StyledBackgroundProps = {
+  backgroundColor: string;
+  backgroundImageJpg: string;
+  backgroundImageWebp: string;
 };
 
-const HeroLayout = (props: React.HTMLAttributes<HTMLDivElement> & HeroLayoutProps) => {
-  const { sxBackground, children } = props;
+const StyledBackground = styled(Box)<StyledBackgroundProps>`
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: -2;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-color: ${(props) => props.backgroundColor};
+  background-image: url(${(props) => props.backgroundImageJpg});
+  background-image: url(${(props) => props.backgroundImageWebp});
```

Comment:
Main improvement here. Not sure you can have this kind of browser-overriding action with Material's `sx`.